build/generate-emoji.go

```diff
@@ -214,8 +214,7 @@ const hdr = `
 
```

Comment:
The `2020` three lines above is now outdated

Reply:
But we should probably fix this on main rather then on this branch.

Reply:
Yup, this is only a backport